doc/guides/west/build-flash-debug.rst

```diff
@@ -157,6 +159,10 @@ You can :ref:`configure <west-config-cmd>` ``west build`` using these options.
    * - ``build.board-warn``
      - Boolean, default ``true``. If ``false``, disables warnings when
        ``west build`` can't figure out the target board.
+   * - ``build.generator``
+     - String, default ``Ninja``. The `CMake Generator`_ to use to create a
```

Comment:
Do you need quotes in `.westconfig`? Because some of the genertor names have spaces in them like "Unix Makefiles".

Reply:
Values with spaces are natively supported by the Python config library.

Reply:
Ah, but there is a bug; it should be config_get('generator', ...), Will fix, thanks!